MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/MapMatchingActivity.java

```diff
@@ -181,16 +186,16 @@ public void onResponse(Call<MapMatchingResponse> call, Response<MapMatchingRespo
             // Convert the map matched response list from position to latlng coordinates.
             // By default, the SDK uses MapMatchingCriteria.GEOMETRY_POLYLINE_6, therefore
             // you need Constants.PRECISION_6 for the decode to be right
-            String geometry = response.body().getMatchings().get(0).getGeometry();
+            String geometry = response.body().matchings().get(0).geometry();
             List<Position> positions = PolylineUtils.decode(geometry, Constants.PRECISION_6);
```

Comment:
This needs to be `Point` right?

Reply:
`PolylineUtils.decode` returns `List<Position>`